e2e/install_test.go

```diff
@@ -522,15 +522,44 @@ func TestSingleNodeUpgradePreviousStable(t *testing.T) {
 		t.Fatalf("fail to check installation state: %v: %s: %s", err, stdout, stderr)
 	}
 
-	appUpgradeVersion := fmt.Sprintf("appver-%s-upgrade", os.Getenv("SHORT_SHA"))
+	appUpgradeVersion := fmt.Sprintf("appver-%s-noop", os.Getenv("SHORT_SHA"))
 	testArgs := []string{appUpgradeVersion}
 
 	t.Logf("%s: upgrading cluster", time.Now().Format(time.RFC3339))
 	if stdout, stderr, err := tc.RunPlaywrightTest("deploy-upgrade", testArgs...); err != nil {
 		t.Fatalf("fail to run playwright test deploy-app: %v: %s: %s", err, stdout, stderr)
 	}
 
-	t.Logf("%s: checking installation state after upgrade", time.Now().Format(time.RFC3339))
+	// use upgraded binaries to run the commands
+	// TODO: this is a temporary workaround and should eventually be a feature of EC
+
+	t.Logf("%s: downloading embedded-cluster %s on node 0", time.Now().Format(time.RFC3339), appUpgradeVersion)
+	line = []string{"vandoor-prepare.sh", appUpgradeVersion, os.Getenv("LICENSE_ID"), "false"}
```

Comment:
Why do you need to call vandoor-prepare here? Do we actually call the on-host embedded-cluster binary in our installation state checks?

For that matter, do we not upgrade that binary in the v2 release process anymore?

Reply:
oh we don't do it in online do we 🤦 

still raises the question of "what is it used for"

Reply:
We check the kots version with the kots cli and we need to update it or it throws an error that the cli version doesn't match the api version

Reply:
Without this i get error

```
          • KOTS CLI version 1.124.0 does not match API version 1.124.1. To update, run:
```

Reply:
i was wrong. removed